### PR TITLE
DS-9749 - Close gaps between chapter 4 of upstream & downstream 'Working on data science projects' guide

### PR DESCRIPTION
**Pull request by jbyrne-redhat**
Validated output via local builds of the ODH website.

### CODE REVIEW

File: modules/deleting-a-model-server.adoc [hunk 1]
```diff
@@ -4,31 +4,35 @@
 = Deleting a model server
 
 [role='_abstract']
-You can delete model servers that you have assigned to host your data science projects. This enables you to remove model servers that you no longer require. If you delete a project's model server, models hosted by the server are then unavailable for applications to use. 
-ifndef::upstream[]
-Red Hat recommends that you advise the users of these models before deleting the model server. 
-endif::[]
+When you no longer need a model server to host models, you can remove it from your data science project. 
+
+NOTE: When you remove a model server, you also delete the models that are hosted on that model server and they are no longer available to applications.
```
Comment (chtyler) [inline modules/deleting-a-model-server.adoc:9]: Regarding the note here:

"... you also delete the models that are hosted on that model server..." 

I assume that you mean that, any models hosted on the deleted model server are automatically deleted, and are therefore no longer available to applications. 

Also, the end of the note says "...and they are no longer available to applications."

File: modules/deleting-a-model-server.adoc [hunk 2]
```diff
@@ -4,31 +4,35 @@
...
 
 .Prerequisites
-* You have logged in to {productname-long}.
-* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, `rhods-users`) in OpenShift.
 * You have created a data science project and an associated model server.
+* You have notified the users of the applications that access the models that the models will no longer be available.
```
Comment (chtyler) [inline modules/deleting-a-model-server.adoc:13]: "are no longer available"?

**Review by chtyler — APPROVED** [review]
Only a couple of minor comments. Looks great otherwise!

File: modules/deleting-a-model-server.adoc (hunk 1 above)
Comment (jbyrne-redhat) [reply modules/deleting-a-model-server.adoc:9]: I didn't write the original text, but yes, I believe that's the intended meaning.

**Comment by jbyrne-redhat** [comment]
> Only a couple of minor comments. Looks great otherwise!

Thanks for reviewing!

**Pull request merged**